### PR TITLE
Spanish language

### PR DESCRIPTION
**Pull request by rcutanda**
Based on the informal style spoken in Spain

### CODE REVIEW

**Comment by aledeg** [comment]
Looks good to me. I don't speak Spanish though.

**Comment by Alkarex** [comment]
Thank you @rcutanda 👍 
Could you please:
* add your name to https://github.com/FreshRSS/FreshRSS/blob/dev/CREDITS.md (only if you want)
* add a line to the changelog https://github.com/FreshRSS/FreshRSS/blob/dev/CHANGELOG.md

**Comment by Alkarex** [comment]
Thanks! One can also push more commits to the same pull-request, to keep them grouped
https://github.com/FreshRSS/FreshRSS/pull/1634
https://github.com/FreshRSS/FreshRSS/pull/1635

File: app/i18n/es/gen.php [hunk 1]
```diff
@@ -0,0 +1,186 @@
...
+	'lang' => array(
+		'cz' => 'Čeština',
+		'de' => 'Deutsch',
+		'en' => 'English',
+		'fr' => 'Français',
+		'it' => 'Italiano',
+		'nl' => 'Nederlands',
+		'ru' => 'Русский',
+		'tr' => 'Türkçe',
+		'zh-cn' => '简体中文'
```
Comment (Alkarex) [inline app/i18n/es/gen.php:124]: Syntax error fixed in https://github.com/FreshRSS/FreshRSS/pull/1636

**Comment by rcutanda** [comment]
I'm sorry about the multiple pulling request. I am new to Github contributions. I will take it into account for the next time. Thanks for your patience. Bests.

**Comment by Alkarex** [comment]
No worries @rcutanda and thanks again :-)

**Pull request merged**